synapse/models/economic.py

```diff
@@ -39,7 +45,8 @@
                 ),
                 'doc': 'A purchase event.'}),
 
-            ('econ:receipt:item', ('guid', {}), {
+            ('econ:lineitem', ('guid', {}), {
+                'prevnames': ('econ:receipt:item',),
```

Comment:
Yeah it looks like currently prevnames is only picked up from prop/form, not the types, might need to move some of these or make it also check for prevnames on types

Reply:
I don't think it can be declared in the form ( no dict ) right?

Reply:
Forms still have an info dict